Dockerfile

```diff
@@ -1,38 +1,36 @@
 # To run: docker run -v /path/to/wsgi.py:/var/www/peregrine/wsgi.py --name=peregrine -p 81:80 peregrine
 # To check running container: docker exec -it peregrine /bin/bash
 
-FROM quay.io/cdis/python-nginx:pybase3-1.4.1
+FROM quay.io/cdis/python:3.7-alpine as base
```

Comment:
why stop using our base image?

peregrine is failing to come up:
```
$ kubectl logs peregrine-deployment-5857c9c85c-9mzbt 
/bin/sh: /dockerrun.sh: not found
```

Reply:
I don't know enough about our docker images. I assumed that was the preferred image for 3.7 and was trying to bump Python while I was here, but it was too much scope creep and broke things like you pointed out. I reverted that back to 3.6.